origarm_uw/Seg_Pre[4] segment